src/page-modules/contact/components/form/input.tsx

```diff
@@ -42,7 +44,11 @@ export const Input = ({
     >
       <div className={style.label_container}>
         <label>
-          <Typo.span textType="body__primary">{label}</Typo.span>
+          <Typo.span
+            textType={disabled ? 'body__primary--strike' : 'body__primary'}
```

Comment:
Ønsker vi egentlig å bruke `strike` her? Det har vi ikke gjort for noen andre inputs som er disabled

Reply:
`Strike` er kanskje ikke riktig å bruke, men jeg tenker at det bør være en fargeendring i label for å signalisere at input feltet er disabled. Fant ikke et disabled alternativ i textType, men det finnes kanskje et slikt alternativ?